They've added "-fuse-ld=gold" WHICH IS NOT EVEN SUPPORTED IN UPSTREAM GCC, WTF??!!!

--- a/src/3rdparty/webkit/Source/common.pri.orig	2014-03-08 10:55:37.029604522 +0100
+++ b/src/3rdparty/webkit/Source/common.pri		2014-03-08 10:55:44.884604873 +0100
@@ -3,13 +3,6 @@
 contains(JAVASCRIPTCORE_JIT,yes): DEFINES+=ENABLE_JIT=1
 contains(JAVASCRIPTCORE_JIT,no): DEFINES+=ENABLE_JIT=0
 
-linux-g++ {
-isEmpty($$(SBOX_DPKG_INST_ARCH)):exists(/usr/bin/ld.gold) {
-    message(Using gold linker)
-    QMAKE_LFLAGS+=-fuse-ld=gold
-}
-}
-
 # We use this flag on production branches
 # See https://bugs.webkit.org/show_bug.cgi?id=60824
 CONFIG += production
